port/unix_include/omrportpg.h

```diff
@@ -86,6 +86,7 @@ typedef struct OMRPortPlatformGlobals {
 	uintptr_t vmem_pageSize[OMRPORT_VMEM_PAGESIZE_COUNT]; /** <0 terminated array of supported page sizes */
 	uintptr_t vmem_pageFlags[OMRPORT_VMEM_PAGESIZE_COUNT]; /** <0 terminated array of flags describing type of the supported page sizes */
 	uint32_t sysinfoControlFlags;
+	uintptr_t isRunningInContainer;
```

Comment:
The name suggests this is a boolean value, which it is not. Please choose a less misleading name.

Reply:
Changed it to `PPG_processInContainerState`. Please let me if there is a better name.